docs/remote_inference_blueprints/amazon_rekognition_connector_blueprint.md

```diff
@@ -0,0 +1,259 @@
+# Amazon Rekognition Detect Text connector blueprint
+
+Amazon Rekognition is a computer vision service that can analyze images and videos. This tutorial shows how to create an OpenSearch connector for Amazon Rekognition's Detect Text API, which can detect and extract text from images.
+
+## Detect text with DetectText API
+
+This tutorial demonstrates how to create an OpenSearch connector for an Amazon Rekognition model that detects text in images using the DetectText API.
+
+Note: This functionality is available in OpenSearch 2.14 or later.
+
+### Step 1: Add the connector endpoint to trusted URLs
+```json
+PUT /_cluster/settings
+{
+  "persistent": {
+    "plugins.ml_commons.trusted_connector_endpoints_regex": [
+      "^https://rekognition\\..*[a-z0-9-]\\.amazonaws\\.com$"
+    ]
+  }
+}
+```
+
+
+### Step 2: Create a connector for Amazon Rekognition
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "rekognition: detect text",
+  "description": "my test connector",
+  "version": "1.0",
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "your_access_key",
+    "secret_key": "your_secret_key",
+    "session_token": "your_session_token"
+  },
+  "parameters": {
+    "region": "us-west-2",
+    "service_name": "rekognition"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "headers": {
+        "content-type": "application/x-amz-json-1.1",
+        "X-Amz-Target": "RekognitionService.DetectText"
+      },
+      "url": "https://${parameters.service_name}.${parameters.region}.amazonaws.com",
+      "request_body": "{ \"Image\": { \"Bytes\": \"${parameters.image_bytes}\" }  } "
+    }
+  ]
+}
+``` 
+Sample response:
+```json
+{
+    "connector_id": "o52l5pMB6Ebhud5_ypxu"
```

Comment:
As I look at other tutorials within the same folder Im seeing they follow the format of showing the connector id and model id. Maybe we should make them placeholders? such as `your_connector_id`

What do you think?